docs/using/flowpipe/mods.md

```diff
@@ -70,6 +70,31 @@ applicable (if left empty, will default to `*`). When finished, click **Install
 
 After the mod is installed, you will be prompted to [set its variables](#managing-mod-variables).
 
+## Installing a Mod from an Archive File
+
+You can install an archived mod by selecting **Install from mod archive** option. This will allow you to upload your own archived mod file subject to the following rules:
+
+- The mod archive file must be of `.zip` format.
```

Comment:
I had suggested that we remove the .tar option to simplify the instructions and only provide docs for .zip and .tar.gz, but it looks like you further simplified it as only .zip  ?  I think it works pretty well and it makes the example simpler, but should we mention that you can upload tar.gz or zip, even though we only provide the .zip example?